src/app/myProfile/profileRead/ProfileRead.tsx

```diff
@@ -40,6 +55,16 @@ const ProfileRead = (props: ProfileReadProps) => {
           </Button>
```

Comment:
there is a logout button which is not functioning, could you either remove it or add a logout function to it?